plugins/capacity.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: capacity
+spec:
+  version: "v0.1.1"
+  homepage: https://github.com/akrzos/kubeSize
+  shortDescription: "Get cluster size and capacity"
+  description: |
+    Exposes size and capacity data for Kubernetes clusters
```

Comment:
It's not clear to me what size and capacity refers to. Number of cores? Memory limit? Number of nodes?
Please be a bit more specific about what numbers you show.

Reply:
Sure I will improve the description.  The main idea was to help provide quick sensible numbers that can determine if you could fit a pod/application based on its schedule-able requirements on the cluster or worker node space.

Reply:
I think that a plugin that determines if you can fit an application with certain requirements into a cluster is a very good idea. However, the approach you take can be improved.

At the moment, your plugin shows certain capacity metrics in the cluster. So the users need to look at their own manifests to compare these numbers and check if it fits or not. But your plugin could do this check for the user!

So imagine you had a plugin called `will-it-fit`. You run `$ kubectl will-it-fit my-deployment.yaml`, and the plugin says `yes` or `no`. Isn't that simple? This will be much more useful and a much better fit than having yet another capacity/cluster-stats plugin.